src/components/responsive-hero-image.js

```diff
@@ -1,9 +1,10 @@
 import React from "react";
-import { ResponsiveImage } from './responsive-image';
-import omit from '@babel/runtime/helpers/objectWithoutProperties';
+import { ResponsiveImage } from "./responsive-image";
+import omit from "@babel/runtime/helpers/objectWithoutProperties";
+import get from "lodash/get";
 
 /**
- * This component takes is a wrapper over {@link ResponsiveImages}, which accepts a story and returns the hero image. By default, it picks the alt text from the headline.
+ * This component takes in a wrapper over {@link ResponsiveImages}, which accepts a story, if the story data is present it returns the story data else if the story data is not present and the alternate data is present it returns alternate data. By default, it picks the alt text from the headline.
  *
```

Comment:
could we rephrase this? not clear what you mean here
IMO keep it simple, just tell that it's a wrapper around `ResponsiveImages`, it picks that from the story. If story isn't present, it'll pick the image from alternate story. And explain what's alternate story

Reply:
Can change it to this:
```
This component is a wrapper over {@link ResponsiveImages}. It accepts story as prop and renders the story's hero image. If hero image is absent, it picks up the alternate hero-image. It picks the headline as alt text
```